src/Products/GenericSetup/interfaces.py

```diff
@@ -535,9 +550,27 @@ def runAllImportStepsFromProfile(profile_id, purge_old=None,
         o Unless 'ignore_dependencies' is true this will also import
           all profiles this profile depends on.
 
+        o 'archive' is a tarball with files to import.
+          Note that metadata.xml is never read, so you cannot use dependencies.
+
         o 'blacklisted_steps' can be a list of step-names that won't be
           executed. Use with special care and only for special cases.
 
+        o 'dependency_strategy' is an integer describing what do we do with
+          already applied dependency profiles.  The default is to apply new
+          profiles (of course), and upgrade outdated ones.
+          See 'DEFAULT_DEPENDENCY_STRATEGY' in 'tool.py'
+
+        o 'path' is the path to a directory with files to import.
+          This avoids the need to register a full profile, when you only need
+          to import a few files once in an upgrade step.
+          It is best to use an absolute path, otherwise it would depend on
+          your working directory.
+          Note that metadata.xml is never read, so you cannot use dependencies.
+
+        o First the code should find a profile via the 'profile_id'.
```

Comment:
Docstrings in interfaces (always) imply  `should`. I suggest a different phrasing - something like: the profile to be imported is determined from *profile_id*, *archive* or *path* (the first with a value different from 'None').

Reply:
Fixed.